src/NHibernate/Mapping/ByCode/IComponentMapper.cs

```diff
@@ -31,4 +33,29 @@ public interface IComponentAttributesMapper<TComponent> : IEntityPropertyMapper
 
 	public interface IComponentMapper<TComponent> : IComponentAttributesMapper<TComponent>, IPropertyContainerMapper<TComponent>
 	{}
-}
\ No newline at end of file
+
+	public static class ComponentAttributesMapper
+	{
+		// 6.0 TODO: Move to IComponentAttributesMapper
+		public static void LazyGroup(this IComponentAttributesMapper mapper, string name)
+		{
+			if (mapper is ComponentMapper component)
+			{
+				component.LazyGroup(name);
+			}
+		}
```

Comment:
I was wondering if it should throw when it is not a `ComponentMapper`. But it seems the pattern for not supported properties is to be just no-op, in mapping ByCode. So well, let it be no-op here too.